pkg/tests/constants.go

```diff
@@ -0,0 +1,31 @@
+package constants
+
+import (
+	"io/ioutil"
+	"os"
+)
+
+var (
+	PromscaleExtensionVersion   string
+	PromscaleExtensionContainer string
+)
+
+func init() {
+	file, err := os.Open("../../../EXTENSION_VERSION")
```

Comment:
I'd recommend another approach. To avoid all the file and parsing code we can just have a const defined with the version and inject that const into env var in the init stage.

Reply:
I don't think that I understand your suggestion. The const would be in go code?

Reply:
Yep. Thinking again that would actually mean that we would need to have env defined in the CI yamls as well which is not great...